core/src/main/java/edu/wpi/grip/core/PreviousNext.java

```diff
@@ -0,0 +1,19 @@
+package edu.wpi.grip.core;
+
+
+/**
+ * An Object that can switch its value.
+ */
+public interface PreviousNext {
```

Comment:
I don't think this interface is useful at all for us.  It one implementation in this module and zero uses, and it has one use in the UI module.  Just use the concrete class.